modules/ROOT/pages/async-scope-reference.adoc

```diff
@@ -47,13 +47,39 @@ Note that even though the Async scope receives a copy of the Mule message, the p
 
 In other words, if the payload of your message is a mutable object (for example, a bean with different fields in it) and a message processor in your Async scope changes the value of one of the fields, the message processors outside of the Async scope see the changed values.
 
-This example shows the XML for the Async scope.
+
+== Example Async Scope Configuration
+
+The following XML snippet shows an example configuration of an Async scope inside an application flow. The Async scope contains a `file:read` operation that executes asynchronously once triggered, while the flow continues processing the next operation in the flow which is `http:request`:
 
 [source,xml,linenums]
 ----
-<async doc:name="Async">
-  <!-- One or more processors here -->
-</async>
+<!-- Main application flow -->
+<flow name="myMainFlow" >
+  <!-- HTTP Listener as message source -->
+  <http:listener doc:name="Listener" />
+  <!-- A Transform operation that executes as part of the main flow -->
+  <ee:transform doc:name="Transform Message" >
+    <ee:message >
+      <ee:set-payload >
+        <![CDATA[%dw 2.0
+        output application/json
+        ---
+        payload]]>
+      </ee:set-payload>
+    </ee:message>
+  </ee:transform>
+    <!-- The Async scope executes its message processors in a different thread, while the main flow continues its execution -->
+    <async doc:name="Async" >
+      <!-- A Try scope to handle errors, because the async scope does not inherit the error strategy from the flow --> 
+      <try doc:name="Try" >
+        <!-- After this operation finishes executing, the next processors in the Async scope starts its execution -->
```

Comment:
There is only one processor in the example. The comment may look out of place. And also it looks it doesn't take the try scope into account.